src/tangle.ts

```diff
@@ -50,6 +50,14 @@ export class Client<T> {
         this.context = this._registerContext();
 
         this._transient = this._register();
+
+        if (this._isBus) {
+            // broadcast previous state to newly connected clients
+            this.context.pipe(
+                debounceTime(100),
```

Comment:
Seems like a magic number for me. Is there any reason we pick `100`? Is it worth making this configurable?

Reply:
yep, i'm planning to move all timeouts/thresholds to an options object in the next round of changes ahead of cutting a release.

Reply:
Btw, there's a few places where a desire for config applies. Planning to review the diff added with these PRs.

Reply:
@christian-bromann picked up work here again. Do you think we should default assume that there's no lazy resolution or keep a low timeout, e.g. 100ms? I already changed the implementation to expect a minimum of `this.providers.length` of clients to connect (please see https://github.com/stateful/tangle/commit/8bf8ad0c3bcb69017574b0b32b4d33b43dbd882b)? Either way, the timeout is going to be configurable. Just wondering if it should be >0.